particles/Processing/Dispose.particle

```diff
@@ -0,0 +1,5 @@
+import './schemas/MlResource.schema'
+
+particle Dispose in './js/Dispose.js'
+  in * {Number ref} resource
```

Comment:
I like the anonymous inline definition for the resource. This is a pattern I discovered late and wish I had used more.

Reply:
This particular usage is important, because there are multiple **unique** TfThing schemas. This particular pattern matches *any* schema that contains `{Number ref}` so it can match the various types.